test/win32/test_registry.rb

```diff
@@ -12,18 +12,38 @@
 if defined?(Win32::Registry)
   class TestWin32Registry < Test::Unit::TestCase
     COMPUTERNAME = 'SYSTEM\\CurrentControlSet\\Control\\ComputerName\\ComputerName'
-
-    private def backslachs(path)
-      path.gsub("/", "\\")
+    TEST_REGISTRY_PATH = 'Volatile Environment'
+    TEST_REGISTRY_KEY = 'ruby-win32-registry-test-<RND>'
+
+    # Create a new registry key per test in an atomic way, which is deleted on teardown.
+    #
+    # Fills the following instance variables:
+    #
+    #   @test_registry_key - A registry path which is not yet created,
+    #                        but can be created without collisions even when running
+    #                        multiple test processes.
+    #   @test_registry_rnd - The part of the registry path with a random number.
+    #   @createopts        - Required parameters (desired, opt) for create method in
+    #                        the volatile environment of the registry.
```

Comment:
The above two are per test, but this options seems invariant.
Why not a constant?